addons/binding/org.openhab.binding.dwdunwetter/README_de.md

```diff
@@ -0,0 +1,103 @@
+# DwdUnwetter Binding
+
+Binding zur Abfrage von aktuellen Unwetterwarnungen des Deutschen Wetterdienstes via [DWD Geoserver](https://maps.dwd.de/geoserver/web/ "DWD Geoserver")
```

Comment:
The URL part behind the space seems unnecessary?

```suggestion
Binding zur Abfrage von aktuellen Unwetterwarnungen des Deutschen Wetterdienstes via [DWD Geoserver](https://maps.dwd.de/geoserver/web/)
```

Reply:
Done